src/main/kotlin/ecommerce/cart/controller/CartController.kt

```diff
@@ -0,0 +1,64 @@
+package ecommerce.cart.controller
+
+import ecommerce.auth.annotation.LoggedMember
+import ecommerce.auth.model.Member
+import ecommerce.cart.model.Cart
+import ecommerce.cart.model.CartDTO
+import ecommerce.cart.service.CartService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/cart")
+class CartController(
+    private val cartService: CartService,
+) {
+    @GetMapping("/")
```

Comment:
```suggestion
    @GetMapping
```

I think `"/"` is a bit redundant here 😄

Reply:
Ok! Actually I realized that I was not using that one, so I deleted it. But I fixed others with similar redundance